### ISSUE TITLE
Unloading Gun

### ISSUE THREAD

**Issue by Zopfco** [New-Chance-RP/Feedback#621]
**A way to reclaim or share ammo**

Drag & drop to "Use" it & unload?  GTA/5M seems weird about ammo.  Magazines that work more like a low-cap fuel can you can stack?

This function might come in handy later with parts or tools for gun customization.  Magazines would also balance slower reloads for the pump shotgun.

**Comment by Mubbb** [comment]
Yes please. It hurts my soul to have 2 pistols, store one and it steal all the ammo. Tie ammo to the weapon pls ty
